### DISCUSSION TITLE
Add "Follow this Game" button to posts

### DISCUSSION THREAD

**Discussion by mikekasprzak** [ludumdare/ludumdare#2062] [Ideas]
While browsing the main feed, you will see pictures of games that catch you eye.

![screenshot from 2018-12-01 18-45-28](https://user-images.githubusercontent.com/1570313/49334052-71a15500-f599-11e8-9871-2b2282371436.png)

It would be good if you could bookmark this game for later. Then when it's time to play+rate games, you have a list of bookmarks, plus your random selection.

**Comment by local-minimum** [comment]
Is this a partial feature of #961 ? Aslo I don't see how this would work since the main feed aren't games but posts about games (perhaps) or do you mean that bookmarking a post on someones game on say LD44 would bookmark their game rather than the post?

Further is this simply a duplicate? Maybe close in that case.
